shesha-reactjs/src/components/calendar/hooks.ts

```
@@ -0,0 +1,139 @@
+    Promise.all(
+      layerWithMetadata?.map(
+        (item) =>
+          new Promise(async (resolve, reject) => {
+            const filter = await evaluateFilters(item, formData, globalState, item.metadata);
+
+            const evalCustomUrl = evaluateString(item.customUrl, { data: formData, globalState });
+
+            refetch(getCalendarRefetchParams({ ...item, customUrl: evalCustomUrl, overfetch: item.overfetch }, filter))
+              .then(resolve)
+              .catch(reject);
+          }),
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)